awx/ui/client/src/notifications/notificationTemplates.form.js

```diff
@@ -261,6 +261,69 @@ export default ['i18n', function(i18n) {
                 subForm: 'typeSubForm',
                 ngDisabled: '!(notification_template.summary_fields.user_capabilities.edit || canAdd)'
             },
+            grafana_url: {
+              label: i18n._('Grafana URL'),
+              type: 'text',
+              awPopOver: i18n._('The base URL of the Grafana server - the /api/annotations endpoint will be added automatically to the base Grafana URL.'),
+              placeholder: 'https://grafana.com',
+              dataPlacement: 'right',
+              dataContainer: "body",
+              awRequiredWhen: {
+                  reqExpression: "grafana_required",
+                  init: "false"
+              },
+              ngShow: "notification_type.value == 'grafana' ",
```

Comment:
Probably want a `===` instead of a `==` for these.

Reply:
Actually, it looks like this is also how the other notification types are configured - so maybe not. If it works 👍

Reply:
Yeah i just copied from the other notification types... Works 👍